customize generatos lookup path

tborychowski opened the pull request:
This will allow us to call `yeoman-ui` command (from another extension) and pass the path, where to look for generators, e.g.:
```ts
    vscode.commands.executeCommand('loadYeomanUI', {
        lookupPath: 'custom/path/to/generators/'
    });

```

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/SAP/yeoman-ui?pullRequest=222) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/SAP/yeoman-ui?pullRequest=222) before we can accept your contribution.<br/><hr/>**Tomasz Borychowski** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/SAP/yeoman-ui?pullRequest=222) it.</sub>

tborychowski commented:
I added some tests of the new code.
However, there are big parts (e.g. in `extension.ts`) that are not tested.
I added some code to these parts, and I have also no idea how to test these parts ;-).

So sorry for "tweaking" your branch coverage threshold.
If you have a better way for me to test them, please let me know.

tborychowski commented:
I had to create a new PR because I pushed these commits with the wrong email.
Please see the #223.
